README.md

```diff
@@ -1,2 +1,17 @@
 # switchboard-samples
 Sample senders and receivers to be used with the Switchboard project. https://github.com/felixlapierre/switchboard
+
+# Requirements
+* ffmpeg
```

Comment:
Could we include the link for ffmpeg [https://ffmpeg.org/](https://ffmpeg.org/)? I orignially thought this requirement referred to the ffmep-python package.

Reply:
Good point, I'll also include the version